caikit/interfaces/nlp/data_model/classification.py

```diff
@@ -30,6 +30,13 @@
 
 log = alog.use_channel("DATAM")
 
+@dataobject(package=NLP_PACKAGE)
+class InputWarnings(DataObjectBase):
+    """Input Warning data object, which returns a reason and message associated with warnings 
+    to issue to a user that causes errors (such as failed text generation)
+    """
+    id: str
+    message: str
```

Comment:
These should probably be annotated with field numbers, looks like we're making sure to do that everywhere else